Rock.Encryption/Crypto.cs

```diff
@@ -190,6 +192,150 @@ public static IDecryptor GetDecryptor(object keyIdentifier)
             return Current.GetDecryptor(keyIdentifier);
         }
 
+#if ROCKLIB
+        /// <summary>
+        /// Asynchronously encrypts the specified plain text.
+        /// </summary>
+        /// <param name="plainText">The plain text.</param>
+        /// <returns>The encrypted value as a string.</returns>
+        public static Task<string> EncryptAsync(string plainText)
```

Comment:
Why overload instead of optional parameter?

Reply:
Mainly to match the synchronous API. However, there isn't a good reason why the synchronous API doesn't use an optional parameter.